pkgs/applications/window-managers/sway/default.nix

```diff
@@ -44,7 +47,6 @@ stdenv.mkDerivation rec {
   ];
 
   mesonFlags = [
-    "-Ddefault-wallpaper=false"
```

Comment:
I also considered `"-Ddefault-wallpaper=${boolToString isNixOS}"` but I think the best approach might be to add a `wallpaper` output (and add that to the NixOS module - which makes it still difficult to override but 5 MB are hopefully not that bad - or we keep it as it is).

Reply:
FWIW Arch installs them by default so I guess this should actually be fine: https://archlinux.org/packages/community/x86_64/sway/